app/config/DocumentationController.scala

```diff
@@ -18,17 +18,16 @@ package config
 
 import controllers.Assets
 import definition.ApiDefinitionFactory
-import javax.inject.{Inject, Singleton}
-import play.api.http.HttpErrorHandler
 import play.api.libs.json.Json
 import play.api.mvc.{Action, AnyContent, ControllerComponents}
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
+import javax.inject.{Inject, Singleton}
+
 @Singleton
 class DocumentationController @Inject()(apiDefinition: ApiDefinitionFactory,
                                         cc: ControllerComponents,
-                                        assets: Assets,
-                                        errorHandler: HttpErrorHandler)
+                                        assets: Assets)
```

Comment:
Why was the errorHandler variable was removed?

Reply:
It's unused